### PR TITLE
implement admin module

### PR DESCRIPTION
**Pull request by big-kahuna-burger**
Implemented admin module and fixed some typos.

### CODE REVIEW

File: modules/admin.js [hunk 1]
```diff
@@ -1,4 +1,58 @@
...
+
+exports.declineVendor = function(vendorId){
+	var update = { status : "declined" };
+	return exports.updateVendor(vendorId, update)
+		.then(function(response){
+			return exports.getVendorWithId(response.body[0].id);
+		});
+};
+
+exports.getVendorWithId = function(vendorId) {
```
Comment (amirsabersharifi) [inline modules/admin.js]: Is there any specific reason to define this function here while there is same function in vendor module?

File: modules/admin.js [hunk 2]
```diff
@@ -1,4 +1,58 @@
...
+        uri: postgrestUrl + '/Vendors?id=eq.' + vendorId,
+        method: 'GET',
+        json: true,
+        resolveWithFullResponse: true,
+        headers: {
+            'Prefer': 'plurality=singular'
+        }
+    });
+};
+exports.updateVendor = function(vendorId, vendorProperties){
```
Comment (amirsabersharifi) [inline modules/admin.js]: same about this function. There is a similar function on vendor module.

File: modules/admin.js [hunk 3]
```diff
@@ -1,4 +1,58 @@
...
+        method: 'GET',
+        json: true,
+        resolveWithFullResponse: true
+    });
+};
+exports.approveVendor = function(vendorId){
+	var update = { status : "approved" };
+	return exports.updateVendor(vendorId, update)
+		.then(function(response){
+			return exports.getVendorWithId(response.body[0].id);
```
Comment (amirsabersharifi) [inline modules/admin.js]: do you need to get vendor with Id again? since update vendor returns the whole update vendor back.

Comment (big-kahuna-burger) [inline modules/admin.js]: The one that I had doubt about. I'll fix it.

File: modules/admin.js (hunk 1 above)
Comment (big-kahuna-burger) [inline modules/admin.js]: Nope. I can reuse the module.

**Pull request merged**